api-specifications.md

```diff
@@ -69,11 +80,12 @@ List all available courses (students+instructors). Used for ExchangeList.
 ```
 
 ##### Error messages
-(TODO)
+* Login required
```

Comment:
This comment applies to all requests in the API. Should we also have "Authentication failed" or "Permission denied" if the user is not authorized to use a particular request? I see some permission denied messages in the vserver code.

Reply:
I am adding the permission denied messages (the one in parenthesis) just for debugging. They should be removed in final product.